src/Razor/test/Microsoft.AspNetCore.Razor.LanguageServer.Test.Common/CSharpTestLspServerHelpers.cs

```diff
@@ -71,7 +77,7 @@ internal class CSharpTestLspServerHelpers
                     name: "TestDocument" + documentCount,
                     filePath: documentFilePath,
                     loader: TextLoader.From(textAndVersion),
-                    razorDocumentServiceProvider: TestRazorDocumentServiceProvider.Instance);
+                    razorDocumentServiceProvider: new TestRazorDocumentServiceProvider(razorSpanMappingService));
```

Comment:
In some cases, we need a real span mapping service since Roslyn LSP rename calls into it